README.md

```diff
@@ -2,6 +2,15 @@
 
 Wallpapers for CachyOS
 
+#### North
+![abstract](https://github.com/erenyldz89/cachyos-wallpapers/blob/develop-new/usr/share/wallpapers/cachyos-wallpapers/north.png)
```

Comment:
May change to https://github.com/CachyOS/cachyos-wallpapers/blob/develop-new/usr/share/wallpapers/cachyos-wallpapers/north.png